pkg/cmd/flags.go

```diff
@@ -59,7 +59,7 @@ func AddWaitFlag(cmd *cobra.Command, wait *kube.WaitStrategy) {
 	cmd.Flags().Var(
 		newWaitValue(kube.HookOnlyStrategy, wait),
 		"wait",
-		"if specified, will wait until all resources are in the expected state before marking the operation as successful. It will wait for as long as --timeout. Valid inputs are 'watcher' and 'legacy'",
+		"wait until resources are ready (up to --timeout). Values: 'watcher', 'hookOnly' (default), and 'legacy'",
```

Comment:
As "Problem 2: Confusing CLI Mapping" points out, there are two "defaults". The default for when no flag is specified. And the default for when the flag is specified, but no value is provided (the result of convert from a boolean flag)

Thinking about how to represent these. Perhaps?:

```suggestion
		"if specified, wait until resources are ready (up to --timeout). Values: 'watcher' (default), 'hookOnly', and 'legacy'.",
```